api/proto/ci/gcb.proto

```diff
@@ -30,7 +28,4 @@ message GoogleCloudBuildAccount {
   // Google Compute Engine VM, or needs permissions not afforded to the VM
   // it is running on.
   string jsonKey = 4;
-
-  // Fiat permissions configuration.
-  Permissions permissions = 5;
```

Comment:
While there aren't any commands, you can still set it by directly editing the halconfig YAML, so it might make sense to keep this here to avoid breaking users who have done that.

Reply:
🆗 ✅